components/server/src/user/user-controller.ts

```diff
@@ -436,6 +436,23 @@ export class UserController {
                 res.sendStatus(200);
             },
         );
+        router.get(
+            "/auth/frontend-dev",
```

Comment:
I didn't understand why we need this new route

Reply:
What alternative could we use instead? I think I came up with something separate to stay flexible. E.g., do we want to stick with that permission, or sth different? :shrug: Based on that we can try to re-use an existing one.

Reply:
The new route is used by the new `forward_auth` directive in the Caddyfile - Caddy delegates to this new route in order to check if the user is authorized to switch out the dashboard at runtime.

Sorry if I misunderstood your question @svenefftinge but I also had to remind myself how the [forward-auth](https://caddyserver.com/docs/caddyfile/directives/forward_auth) directive works.